### DISCUSSION TITLE
Request: Rolling 7 days instead of 5

### DISCUSSION THREAD

**Discussion by stites** [iSoron/uhabits#883] [Feature Requests] [answered]
Closer in nature to https://github.com/iSoron/uhabits/issues/708 than https://github.com/iSoron/uhabits/discussions/836 / https://github.com/iSoron/uhabits/discussions/51, I would be interested in adjusting the number of rolling days in the main view from 5 to 7 so that I always have a detailed overview of all habits for "the past (rolling) week."

I'm a new user and I think that this simple adjustment would be a more user-friendly for anyone who plans to do weekly reviews of all habits. My current, offline process of review for the past 6-months is identical to reviewing a 7-day tabulation from Loop 2.0 -- so I am not invested in comprehensive visualizations covered in https://github.com/iSoron/uhabits/discussions/51 or https://github.com/iSoron/uhabits/discussions/31. These would, perhaps, be important to more of a habit-tracking power-user (but I am a bit more low-tech).

**Comment by stites** [comment, marked as answer]
Solved via landscape mode!

**Reply by hockeymikey** [reply]
How is landscape a solution?

**Reply by stites** [reply]
Because it solved my issue of only thinking I could see 5 days at a time and now I have 12 days of overview.

**Comment by hiqua** [comment]
Note that you can also scroll horizontally if you swipe the days left.

Honestly I don't like it so much because it's not really discoverable and the swipe is not super smooth, so there's room for improvement, but it's still a decent workaround.

**Reply by hockeymikey** [reply]
The swipe sucks. I think it would be better if it was treated as a list so it reveals/moves them if you run your finger over it and move it compared to a blind swipe IMO but maybe Im off base.

**Reply by stites** [reply]
I agree, the swipe is very bad (just rotating to landscape has been better, for me). It also seems like you should be able to swipe on the checks themselves.

**Comment by hockeymikey** [comment]
I think this is still needed, there is plenty of room for 2 more days and can make the number configurable.
